CONTRIBUTING.md

```diff
@@ -1,95 +1,39 @@
-# How to Contribute
+# Contributing to the InnerSource Patterns
 
-First, checkout our complete list of patterns: [public GitHub](https://github.com/InnerSourceCommons/InnerSourcePatterns#reviewed-patterns-proven-and-reviewed) OR [Google Doc](https://docs.google.com/spreadsheets/d/17KPZdCoquTnYSj03pX4v2vn8lrSYO_6HK20u1cwaLPg/edit#gid=0)
+The goal of the InnerSource patterns working group is to collect, document, and publish InnerSource best practices. To make the best practices easy to comprehend, evaluate, and apply, we codify them in a specific structure - [the patterns format](meta/pattern-template.md). More infos on the working group can be found in our [README.md](README.md) file.
 
-We encourage beginners seeking answers to jump in by creating `donut` patterns (filling in the problem, context, forces and resulting context fields but leaving the solution blank) as a way of asking the InnerSource Commons community for help (to find a proven solution or to brainstorm things to try). 
+For this working group to strive, we welcome your contribution - be it small or huge. 
 
-Anyone can offer reviews and comments for [in-progress patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns/pulls). We encourage experts to pad their experience - these are hoped to become part of an Inner Source handbook one day.
 
-We work together via GitHub, WebEx, Slack, etc. Do not hesitate to join the [#innersourcecommons](https://isc-inviter.herokuapp.com/) or #innersource-patterns Slack channels and ask to be included in the [patterns meetings](/meta/meetings.md) (there is an email list).
+## How to Contribute?
 
-Select one of the following ways to contribute, to learn our workflow:
+Please consider contributing in the following ways:
 
-* A. [Write a new pattern](#a-writing-a-new-pattern)
-* B. [Discuss/Record early ideas](#b-discussing-early-ideas-in-issues)
-* C. [Review existing patterns](#c-interacting-with-patterns-reviews)
-* D. [Take part in our Meetings and Roles](/meta/meetings.md) <-- link to separate doc
+1. **Share your thoughts & spark discussion**. A fresh perspective, an intersting thought, or an early novel idea can spark discussions and yield unforeseeable results. We invite you to share your thoughts on InnerSource-related topics at any time by [creating an issue](https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/new), [via Slack, or our patterns office hours](#how-to-get-in-touch).
```

Comment:
```suggestion
1. **Share your thoughts & spark discussion**. A fresh perspective, an interesting thought, or an early novel idea can spark discussions and yield unforeseeable results. We invite you to share your thoughts on InnerSource-related topics at any time by [creating an issue](https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/new), [via Slack, or our patterns office hours](#how-to-get-in-touch).
```

Reply:
@maxcapraro found a tiny typo but other than this I really like the approach now. Now let's turn it into practise :)